Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/DiscreteConnectivity.asl

```diff
@@ -0,0 +1,101 @@
+/** @file
+  Intel ACPI Sample Code for Discrete Connectivity module

```

Comment:
It is not necessary to add this file for S0ix support.

Reply:
Done removed it, thanks.